build/build.sh

```diff
@@ -377,6 +386,16 @@ for PATCH_TO_APPLY in "${PATCHES_TO_APPLY[@]}"; do
     git -C "${ROOT}/llvm-project" apply "${PATCH_TO_APPLY}" -v
 done
 
+# Setup external projects
+mkdir -p ${EXTERNAL_PROJECT_DIR}
+for I in "${!EXTERNAL_PROJECT_URLS[@]}"; do
```

Comment:
can you use `EXTERNAL_PROJECT_URL` instead of `I` please, and similar below. thanks!